packages/opc-footer/README.md

```diff
@@ -1,112 +1,138 @@
-# This is opc-footer 👋
+# opc-footer component 👋
 
-![Version](https://img.shields.io/badge/version-0.0.1-blue.svg?cacheSeconds=2592000)
+![Version](https://img.shields.io/badge/version-0.0.2-blue.svg?cacheSeconds=2592000)
 ![Build Status](https://travis-ci.org/dwyl/esta.svg?branch=master)
 [![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/1-Platform/op-components/graphs/commit-activity)
 
-opc-footer is a web component based on lit-element. By default it has the light theme and text for copyright.
+A standardized web component based on Lit Element for Red Hat One Platform Footer.
 
-## Install
+# Usage
+### Footer with links:
+* The links needs to have following data type which is defined in `./types.d.ts` file. Below is the mock data example.
+* use the `opcLinkCatagories` setter function to add links.
+```js
+const links = [
+  { "group":"Group name 1",
+    "links":[
+      { "name":"Lorem ipsum", "path":"https://example.com" },
+      { "name":"Lorem ipsum" },
+    ]},
+  { "group":"Group name 2",
+    "links":[
+      { "name":"Lorem ipsum", "path":"https://example.com" },
+      { "name":"Lorem ipsum" },
+      { "name":"Lorem ipsum" },
+  ]},
+];
 
-```sh
-npm install
+const myFooterEl = document.querySelector('opc-footer');
+myFooterEl.opcLinkCatagories = links;
 ```
 
-## Usage
+* The Links with no path fires the custom event called `opc-footer-link:click`, so the user can add custom behavior. for example 
+```js
+  const myFooterElement = document.querySelector('opc-footer');
+  myFooterElement.addEventListener('opc-footer-link:click', (e) => {
+    console.log(e);
+    /// custom behaviors.
+  });
+```
 
-the opc-footer have supports dark theme, by default the light theme is selected. for example
-footer with default theme:
+Default with light theme: 
 
 ```html
 <opc-footer></opc-footer>
 ```
 
-![Demo Footer light with no links](demo/images/Footer-light-no-links.png)
+Dark theme:
+in `html`
+```html
+<opc-footer theme="dark"></opc-footer>
+```
+as well you can change the theme in `js` too by selecting the element reference.
+```js
+const myFooterEl = document.querySelector('opc-footer');
+myFooterEl.theme = 'dark';
+```
 
-here's another with dark
+# Slot
+The `opc-footer` component supports the slot for foot note which is reserved for copyright text. By default it shows the text as Copyright **All Rights Reserved.**
+if you want to change use the slot name as `copyright` for example.
```

Comment:
It should be `You can use "copyright" slot to provide custom text` as shown in below example

Reply:
updates description for slot:

The `opc-footer` component supports the slot for foot note which is reserved for copyright text. By default it shows the text as Copyright **All Rights Reserved.**
You can use `copyright` slot to provide custom text as shown in below example